plugins/lime-plugin-align/src/alignActions.js

```diff
@@ -4,12 +4,13 @@ import {
 	IFACES_LOAD,
 	SIGNAL_GET
 } from './alignConstants';
-import { store } from '../../../src/store';
+// import { store } from '../../../src/store';
 
 export const changeInterface = (iface) => {
-	if (iface === store.getState().align.currentReading.iface) {
-		return { type: 'no_iface' };
-	}
+	// TODO: FIX This store usage
```

Comment:
fix before merge

Reply:
It ended up not being an issue, no fix needed, thanks!